x/auth/client/cli/tx_multisign.go

```diff
@@ -195,6 +201,155 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 	}
 }
 
+func GetMultiSignBatchCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:    "multisign-batch [file] [name] [[signature-file]...]",
+		Short:  "Assemble multisig transactions",
+		Long:   `Assemble a batch of multisig transactions generated by batch sign command.`,
+		PreRun: preSignCmd,
+		RunE:   makeBatchMultisignCmd(),
+		Args:   cobra.MinimumNArgs(3),
+	}
+
+	cmd.Flags().Bool(flagNoAutoIncrement, false, "disable sequence auto increment")
+	cmd.Flags().String(
+		flagMultisig, "",
+		"Address of the multisig account on behalf of which the transaction shall be signed",
+	)
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func makeBatchMultisignCmd() func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		clientCtx := client.GetClientContextFromCmd(cmd)
+		clientCtx, err := client.ReadTxCommandFlags(clientCtx, cmd.Flags())
+		if err != nil {
+			return err
+		}
+
+		txCfg := clientCtx.TxConfig
+		txFactory := tx.NewFactoryCLI(clientCtx, cmd.Flags())
+		if txFactory.SignMode() == signingtypes.SignMode_SIGN_MODE_UNSPECIFIED {
+			txFactory = txFactory.WithSignMode(signingtypes.SignMode_SIGN_MODE_LEGACY_AMINO_JSON)
+		}
```

Comment:
I think we should be explicit here.
1. No default re-assignment
2. if sign mode == DIRECT then return with error.

Reply:
Maybe we could just document this behaviour in the command's long description?

Reply:
Command's description was updated. Hence resolving.